index.html

```diff
@@ -14,55 +14,129 @@
+                    <h2 id="title-line">I'am a Software Developer</h2>
+                    <p id="para">I can help you build a product, feature or website Look through some of my work <br> and experience! If you like what you see <br> and have a project you need coded, don't hestiate to contact me.</p>
+                </div>
+                <ul class="socials-container">
+                    <li class="socials">
+                        <a href="https://github.com/Amrit131" class="links"><img src="images/github.png" alt="github link" class="social-img"></a>
+                    </li>
+                    <li class="socials">
+                        <a href="https://www.linkedin.com/in/mishra-amrit/" class="links"><img src="images/linkedin.png" alt="linkedin link" class="social-img"></a>
+                    </li>
+                    <li class="socials">
+                        <a href="http://angellist.com" class="links"><img src="images/dnt.png" alt="dnt link" class="social-img"></a>
+                    </li>
+                    <li class="socials">
+                        <a href="https://twitter.com/i_ammrit" class="links"><img src="images/tweet.png" alt="Twitter link" class="social-img"></a>
+                    </li>
+                    <li class="socials">
+                        <a href="http://medium.com" class="links"><img src="images/microverse.png" alt="Microverse link" class="social-img"></a>
+                    </li>
+                </ul>
+            </div>
+            <div id="bg-container" class="socials-container"> 
+                <img id="bg-img" src="images/Groupbackground.png" alt="Background card">
             </div>
+            <div class="bg-container"></div>
         </section>
         <section class="works" id="my-works">
-            <div id="header2-container">
-                <h2 id="header-2">My Recent Works</h2>
-                <hr id="line">
-            </div>
-            <div id="gymfit-container">
-                <img id="gymfit-img" src="images/gym-fit.png" alt="Gym Fit Image">
-                <h3 id="header-3">Multi-Post Stories</h3>
-                <p id="stories-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-                <ul class="language" id="tech-btn">
-                    <li class="btn"><button>css</button></li>
-                    <li class="btn"><button>html</button></li>
-                    <li class="btn"><button>bootstrap</button></li>
-                    <li class="btn"><button>Ruby</button></li>
-                </ul>
-                <div id="btn-container">
-                    <button id="btn">See Project</button>
+            <div id="works-container">
+                <div id="header2-container">
+                    <h2 id="header-2">My Recent Works</h2>
+                    <hr id="line">
```

Comment:
It's almost done! but please make sure to fix the extra space between the line and the text.

![image](https://user-images.githubusercontent.com/34779928/194726972-624d8d65-37d6-4d76-b4fa-5fa1151791e9.png)

Reply:
# Implemented

![2nd](https://user-images.githubusercontent.com/87594100/194728048-54e7c82e-027a-417a-b2e0-c6f3156119f7.png)